module/gui/SettingsWidget.py

```diff
@@ -56,126 +73,346 @@ def loadConfig(self):
         layout = self.layout()
         layout.setSizeConstraint(QLayout.SetMinAndMaxSize)
 
-        general = QTabWidget()
-        self.general = general
-
-        plugins = QTabWidget()
-        self.plugins = plugins
-
-        tab = QTabWidget()
-        self.tab = tab
+        self.tab     = QTabWidget()
+        self.general = QTabWidget()
+        self.plugins = QWidget()
+        self.plugins.setLayout(QVBoxLayout())
+        self.pluginsComboBox = QComboBox()
+        self.pluginsMenuCheckBox = QCheckBox(_("Add To Menu"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['QCheckBox' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)